### ISSUE TITLE
Timeline does not work for a Meteorite Landing layer 

### ISSUE THREAD

**Issue by ElenaGallo** [geosolutions-it/geonode-mapstore-client#210]
The timeline is not available for the *Meteorite Landing* layer. [Here](https://app.zenhub.com/files/122484652/2d2591a7-c39c-42dc-b129-f158e0955ed0/download) the original shapefile

**How to reproduce**
- From master.demo opens [this layer](https://master.demo.geonode.org/layers/geonode_master_data:geonode:Meteorite_Landings_from_NASA_Open_Data_Portal)

**Comment by allyoucanmap** [comment]
It seems it's missing the step to set the time dimension on a layer field
![image](https://user-images.githubusercontent.com/19175505/108215657-dbce5200-7131-11eb-9ff7-a62029ad7579.png)

After checking the shape on QGIS it seems the year is encoded with a strange format
![image](https://user-images.githubusercontent.com/19175505/108215860-159f5880-7132-11eb-813a-81376ef58313.png)

@ElenaGallo where did you get that shape file?

**Comment by ElenaGallo** [comment]
@allyoucanmap I downloaded it from mapstore

**Comment by giohappy** [comment]
There's clearly a problem with the encoding of the "year" field.

**Comment by allyoucanmap** [comment]
@ElenaGallo makes me notice that same layer with get feature info returns an ISO string time
![image](https://user-images.githubusercontent.com/19175505/108219734-4a151380-7136-11eb-9cb1-1257b3d34abf.png)

**Comment by giohappy** [comment]
so it might be a problem with the WPS download

**Comment by giohappy** [comment]
I've done some tests. To me the downloaded shapefile looks good, I odn't know why you see those strange values in QGIS @allyoucanmap.

The real issue is that the Geoserver iimport utility assign a java.sql.Timestamp binding to that attribute, and GeoNode doesn't include that type within the [parser for the selection of the attributes](https://github.com/GeoNode/geonode/blob/69bc39de6fb3db8f174f1c16d52c266bb73db65a/geonode/upload/utils.py#L481-L498) eligable for the time support.

![image](https://user-images.githubusercontent.com/571129/108226791-813af300-713d-11eb-8503-b08f3f7c7d17.png)

Maybe the new versions of Geoserver (and Geotools) have changed the way these fields get parsed. 

I will open an issue for this.

**Comment by giohappy** [comment]
https://github.com/GeoNode/geonode/issues/6962

**Comment by giohappy** [comment]
There's something weird with this file. I ws able to see it correctly before, but testing it again I also see the bad encoding in QGIS.
The problem with the encoding alsa happens when downloading from [the MapStore dev server](https://mapstore2.geo-solutions.it/mapstore/#/viewer/openlayers/7306). 

I suspect that this is due to the datetime extension in Geoserver. @tdipisa I don't have access to the Geoserver of the MapStore demo. Could you please confirm it that extension is active?

**Comment by tdipisa** [comment]
@giohappy do you mean the WMTS multidim extension? Yes it is installed in gs-stable.

**Comment by giohappy** [comment]
I was referring to the `-Dorg.geotools.shapefile.datetime` JAVA_OPTS @tdipisa 

Anyway, after further investigations I was able to connect the dots:

1. @ElenaGallo the sample file you have provided has a special encoding for the year field. That encoding is generated by Geoserver when the  `-Dorg.geotools.shapefile.datetime` is activated (true). I don't know where you obtained that file from but it doesn't matter. So **this issue can be closed, since GeoNode doesn't support the creation of a time series using the that special encoding generated by GeoNode**
2. Geoserver is able to convert back the encoding (even if `-Dorg.geotools.shapefile.datetime` is false, as it is by default for a GeoNode setup. This means that with a little work in GeoNode we could add the support for that encoding, and everything would work fine. I have opened an issue for this, but this could be eventually considered an enhancement not a bugifx

If you agre @ElenaGallo we can close this.
